src/main/java/com/graphhopper/converter/api/OpenCageDataEntry.java

```diff
@@ -77,6 +73,20 @@ public OCDComponents() {
         public String village;
         @JsonProperty("hamlet")
         public String hamlet;
+        @JsonProperty
+        public String postcode;
+
+        // Possible Street names TODO: Not sure what tags can be returned here
+        @JsonProperty
+        public String road;
+        @JsonProperty
+        public String pedestrian;
+        @JsonProperty
+        public String path;
+        @JsonProperty
+        public String footway;
+        @JsonProperty
+        public String construction;
```

Comment:
I wans't sure if this works with Jackson, but I was able to move all duplicated entries to an abstract class.